Client/Frontend/Toolbar+URLBar/TabLocationView.swift

```diff
@@ -102,6 +102,10 @@ class TabLocationView: UIView, FeatureFlaggable {
         trackingProtectionButton.addTarget(self, action: #selector(self.didPressTPShieldButton(_:)), for: .touchUpInside)
         trackingProtectionButton.clipsToBounds = false
         trackingProtectionButton.accessibilityIdentifier = AccessibilityIdentifiers.Toolbar.trackingProtection
+        trackingProtectionButton.showsLargeContentViewer = true
+        trackingProtectionButton.largeContentImage = .templateImageNamed(StandardImageIdentifiers.Large.lock)
+        trackingProtectionButton.largeContentTitle = .TabLocationLockButtonLargeContentTitle
+        trackingProtectionButton.accessibilityLabel = .TabLocationLockButtonAccessibilityLabel
```

Comment:
I reused an existing accessibility label. Should I create a separate localized string for the `largeContentTitle`?